Dockerfile

```diff
@@ -3,7 +3,9 @@ RUN apt-get update &&\
     apt-get install -qqy imagemagick optipng &&\
     mkdir /usr/src/app &&\
     rm -rf /var/lib/apt/lists/*
+
 WORKDIR /usr/src/app
 ADD . /usr/src/app
+RUN pip install -r requirements.txt
 
-ENTRYPOINT ["python", "build-assets.py"]
+ENTRYPOINT ["python"]
```

Comment:
My `docker + python` is pretty elementary, but just checking you didn't want to replace this with `build-assets-from-freelex.py, build-assets-from-freelex.py`

Reply:
@mischa-s not in this case, because we now want to let the caller choose which script to run. Docker combines `ENTRYPOINT` with `CMD` to build the command to run, so in this case, we're removing the hardcoded script and leaving CMD blank, so the Makefile can pass the right script to run (Freelex or Signbank)